net-print/npadmin/npadmin-0.8.7-r2.ebuild

```diff
@@ -1,7 +1,8 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
```

Comment:
You'll need to keep the `r1` and change keywords for the `r2` to:
```
KEYWORDS="~amd64 ~ppc ~x86"
```

Committing a `EAPI` bump with some changes directly to stable is usually not allowed.

Further more you can also update the `HOMEPAGE` to use `https` and remove at least `USE=""`. I would also remove `DEPEND=""` (including the comment). However maybe its worth keeping the comment - in that case the empty `DEPEND=""` might need to stay too..